working/2509-private-named-parameters/feature-specification.md

```diff
@@ -395,9 +396,11 @@ with private name *p* in constructor C:
     compile-time error. *If removing the `_` leads to a collision with
     another parameter, then there is a conflict.*
 
-*   Otherwise, the name of the parameter in C is *n*. *If the parameter is
-    named, this then avoids the compile-time error that would otherwise be
-    reported for a private named parameter.*
+*   Otherwise, the name of the parameter in the signature of C is *n*. By "in
```

Comment:
On line 315, it says
> It doesn't allow _any_ parameter to start with underscore …

It continues to say that it's only some, but this first part of the sentence by itself can be read to say "allows no parameter to ...", which is confusing, and makes the reader have to reconsider what they just read. And then reconsider that first part of the sentence when they read the rest. 

Maybe "Not all named parameters …"?

Reply:
You're right, that sentence is confusing. Rewrote. English is weird.